mesonbuild/envconfig.py

```diff
@@ -11,19 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from __future__ import annotations
 
+from __future__ import annotations
 from dataclasses import dataclass
+from enum import Enum
 import platform
 import subprocess
+import sys
 import typing as T
-from enum import Enum
```

Comment:
These changes seem like pointless churn, but in particular I think it makes a lot of sense to have `__future__` be its own "group" the same way stdlib, first-party, and third-party are each their own groups.

Since, the purpose of `__future__` imports is to act as a bytecode compiler cookie rather than actually importing useful objects. 

(Also you really don't want it to get jumbled into stdlib imports and then accidentally moved away from the first line.)

Reply:
I can move the `__future__` to it's own block.